browser/css/notebookbar.css

```diff
@@ -533,11 +533,12 @@
 	text-align: center;
 }
 
-#style1 img ~ .ui-drawing-area-placeholder-container .ui-drawing-area-placeholder {
```

Comment:
hi, thanks for contributing to the project :)
Unfortunately this is not what issue #5053 is about. This particular case is ok. We want to fix only "UNO toolbuttons".
Because currently when we have two buttons with the same command - id will be the same. So we add the number, but it's not guaranted that they will appear in the same order so better to use: `#Command1` -> `.unoCommand`.
So you should check if the element with id `XXXX1` is the toolbutton and has class `unoXXXX`, then it should be fixed.

Reply:
okay @eszkadev , so what if I hit something like `#slide-2 fig` it doesn't have an `uno` class. I have to skip right? if yes, I went through all the CSS files, and I could not find any other.

Reply:
yes `slide-2` is also ok :), then maybe the issue is solved already